libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -61,6 +62,7 @@ class NavigationOptions private constructor(
     val historyRecorderOptions: HistoryRecorderOptions,
     val eventsAppMetadata: EventsAppMetadata?,
     val enableSensors: Boolean,
+    private val copilotOptions: CopilotOptions,
```

Comment:
How are you going to access these options? They are private, so not visible to anyone, is it intentional?

Reply:
That's a great catch! Let me rework it 👍

Reply:
I guess they can be internal?

Reply:
I've just fixed it and re-pushed, ready for another round of 👀